application/helpers/SurveyRuntimeHelper.php

```diff
@@ -1867,4 +1872,15 @@ public function getCurrentQuestionClasses($iQid)
 
         return $aQuestionClass;
     }
+
+    /**
+     * Saves the starting values, if any is set
+     */
+    private function saveStartingValues()
+    {
+        // Don't save the starting values if on Welcome screen
+        if ($this->sSurveyMode != 'survey' && $_SESSION[$this->LEMsessid]['step'] == 0) return;
```

Comment:
Question : what happen if welcome page is false ?

Reply:
if welcome page is false, step will be 1